definitions/AuthorisationComplexType.json

```diff
@@ -0,0 +1,26 @@
+[
+  {"CaseTypeID": "NFD-3686", "CaseFieldID": "applicant1SolicitorOrganisationPolicy", "ListElementCode": "OrgPolicyCaseAssignedRole", "UserRole": "caseworker-caa", "CRUD": "CRU"},
```

Comment:
For testing in preview, the CaseTypeID here needs to include the PR number (rather than just being "NFD"). I think the longer term solution is to either output these files with config generator or write a script that substitutes in the correct CaseTypeID when it copies our custom JSON into the definitions folder